resources/i18n/ja_JP/fdmprinter.def.json.po

```diff
@@ -1560,11 +1560,11 @@ msgstr "インフィルオーバーハング角度"
 
 msgctxt "infill_overlap_mm label"
 msgid "Infill Overlap"
-msgstr "インフィル重複"
+msgstr "インフィルの重なり"
 
 msgctxt "infill_overlap label"
 msgid "Infill Overlap Percentage"
-msgstr "インフィル重複率"
+msgstr "インフィルの重なり (%)"
```

Comment:
Maybe the extra % symbol is not required here ? In the UI this label will be just besides the input field, which also contains the symbol. But it seems you have added it on purpose, so I'll let you decide.

Reply:
Thanks for the quick response.
I added the unit because there is another `インフィルの重なり` (the length of overlap) just below this item.
I have considered `インフィルの重なり率` and `インフィルの重なりの割合`, but both were not natural expressions.

Reply:
Ok, so if you have considered it and explicitely decided that this is the best option, I'm good with it :+1: